browser/playlist/test/playlist_browsertest.cc

```diff
@@ -51,7 +56,7 @@ std::unique_ptr<net::test_server::HttpResponse> HandleRequest(
 
 }  // namespace
 
-class PlaylistBrowserTest : public InProcessBrowserTest,
+class PlaylistBrowserTest : public PlatformBrowserTest,
                             public PlaylistServiceObserver {
```

Comment:
Wow, now I can see we're stepping into cross-platform world. Never have used this before.

Reply:
Let's see this works in CI :)

Reply:
Oh, these test works on Android :) Android CI passed!